_docs/concepts/security/rbac.md

```diff
@@ -0,0 +1,221 @@
+[authorization template](https://github.com/istio/istio/blob/master/mixer/template/authorization/template.proto). The request context
+ contains all the information about the request and the environment that an authorization module needs to know. In particular, it has two parts:
+
+* **subject** contains a list of properties that identify the caller identity, including "user" name/ID, "groups" the subject belongs to,
+or any additional properties about the subject such as namespace, service name.
+* **action** specifies "how a service is accessed". It includes "namespace", "service", "path", "method" that the action is being taken on,
+and any additional properties about the action.
+
+Below we show an example "requestcontext".
+```rule
+   apiVersion: "config.istio.io/v1alpha2"
+   kind: authorization
+   metadata:
+     name: requestcontext
+     namespace: istio-system
+   spec:
+     subject:
+       user: request.auth.principal | ""
+       groups: request.auth.principal | ""
+       properties:
+         service: source.service | ""
+         namespace: source.namespace | ""
+     action:
+       namespace: destination.namespace | ""
+       service: destination.service | ""
+       method: request.method | ""
+       path: request.path | ""
+       properties:
+         version: request.headers["version"] | ""
+```
+
+## Istio RBAC Policy
+
+Istio RBAC introduces ServiceRole and ServiceRoleBinding, both of which are defined as Kubernetes CustomResourceDefinition (CRD) objects.
+
+* **ServiceRole** defines a role for access to services in the mesh.
+* **ServiceRoleBinding** grants a role to subjects (e.g., a user, a group, a service).
+
+### ServiceRole
+
+A ServiceRole specification includes a list of rules. Each rule has the following standard fields:
+* **services**: a list of services.
+* **methods**: HTTP methods or gRPC methods. Note that gRPC methods should be presented in the form of "packageName.serviceName/methodName".
+* **paths**: HTTP paths. It is ignored in gRPC case.
+
+A ServiceRole specification only applies to the **namespace** specified in "metadata" section. The "services" and "methods" are required
+fields in a rule. "paths" is optional. If not specified or set to "*", it applies to "any" instance.
+
+Here is an example of a simple role "service-admin", which has full access to all services in "default" namespace.
+
+```rule
+   apiVersion: "config.istio.io/v1alpha2"
+   kind: ServiceRole
+   metadata:
+     name: service-admin
+     namespace: default
+   spec:
+     rules:
+     - services: ["*"]
+       methods: ["*"]
+```
+
+Here is another role "products-viewer", which has read ("GET" and "HEAD") access to service "products.svc.cluster.local"
+in "default" namespace.
+
+```rule
+   apiVersion: "config.istio.io/v1alpha2"
+   kind: ServiceRole
+   metadata:
+     name: products-viewer
+     namespace: default
+   spec:
+     rules:
+     - services: ["products.svc.cluster.local"]
```

Comment:
nit: should be `products.default.svc.cluster.local`

Reply:
Updated.